lib/clients/BaseClient.py

```diff
@@ -22,8 +22,11 @@ def __init__(self, operation_name, configuration, directory_persistent, director
         self.JOB_NAME = configuration['job_name']
         self.DIRECTORY_PERSISTENT = directory_persistent
         self.DIRECTORY_WORK_LIST = directory_work_list
-        assert len(self.OPERATION) > 0, 'No operation name (backup or restore) given.'
-        assert len(self.DIRECTORY_PERSISTENT) > 0, 'Directory of the persistent volume not given.'
+        self.DIRECTORY_DATA = '/var/vcap/data'
+        assert len(
```

Comment:
unnecessary change detection. Can you please remove them so that they do not appear as change?

Reply:
Okay .... I have used a plugin in my IDE using that on save auto indentation is enabled. That is also recommended for production code. As discussed I'll put in README.md